libs/gperftools/Makefile

```diff
@@ -0,0 +1,80 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gperftools
+PKG_VERSION:=
+PKG_RELEASE:=1
+
+#PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+#PKG_SOURCE_URL:=https://codeload.github.com/$(PKG_NAME)/$(PKG_NAME)/tar.gz/$(PKG_NAME)-$(PKG_VERSION)?
+#PKG_HASH:=83e3bfdd28b8bcf53222c3798d4d395d52dadbbae59e8730c4a6d31a9c3732d8
+
+PKG_SOURCE_URL:=https://github.com/gperftools/gperftools
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_DATE:=2023-07-22
+PKG_SOURCE_VERSION:=42dde1b883b29921ac89234369f88b133821690c
```

Comment:
Do we really need to check out the sources to have the latest version instead of properly tagged version? If you want to have tagged the latest version, just ask the developer.

Reply:
I asked: https://github.com/gperftools/gperftools/issues/1406#issuecomment-1658987755

Reply:
fixed in latest commit